build-tools/debian-metadata/xamarin.android.links

```diff
@@ -0,0 +1,2 @@
+/usr/lib/xamarin.android/xbuild-frameworks/MonoAndroid	/usr/lib/mono/xbuild-frameworks/MonoAndroid
+/usr/lib/xamarin.android/xbuild/Xamarin			/usr/lib/mono/xbuild/Xamarin
```

Comment:
This is viable now, but potentially problematic if we ever get another Xamarin system on Linux.

Though at present I doubt iOS or Mac will exist on Linux, so this *probably* isn't something to worry about...